src/TropicalGeometry/variety.jl

```diff
@@ -323,20 +323,19 @@ end
 #
 ################################################################################
 
-function tropical_variety_zerodimensional(I::MPolyIdeal,nu::TropicalSemiringMap{QQField,QQFieldElem,<:Union{typeof(min),typeof(max)}})
-
-    k,(a,_) = number_field(I)
-    zk = maximal_order(k)
-    p = uniformizer(nu)
-    lp = [x[1] for x = prime_decomposition(zk,p)]
-    ma = representation_matrix(a)
-    mb = representation_matrix(k(lp[1].gen_two*lp[2].gen_two^2))
-    @assert iszero(ma*mb - mb*ma)
-    Qp = PadicField(p, 10)
-    TropVDict = simultaneous_diagonalization([map_entries(Qp, ma),map_entries(Qp, mb)])
-
-    TropVPoints = collect(values(TropVDict))
-    TropVPointsUnique = unique(TropVPointsMults)
+function tropical_variety_zerodimensional(I::MPolyIdeal,nu::TropicalSemiringMap{QQField,ZZRingElem,<:Union{typeof(min),typeof(max)}}; precision::Int=64)
```

Comment:
During triage it was pointed out that this should really called `tropical_variety_zero_dimensional`.

But since this is an existing function, this is not actually a change request for this PR! 

That said, this function is not actually exported, so maybe you can just rename it... but perhaps you advertised it somewhere and people have code using it. In that case the old name could be kept as deprecated alias (using `@deprecate` -- feel free to ask for help if you are not familiar with that mechanism).


But again, that's not a requirement for this PR, but I thought I'd mention it now for completeness and before it is "too late" because this got exported in a release...

Reply:
I've renamed the function.